samples/unit-tests/annotations/annotations-measure/demo.js

```diff
@@ -29,37 +29,53 @@ QUnit.test('#13664 - annotation measure on yAxis', function (assert) {
             }
         ],
 
-        series: [
-            {
-                data: [1, 2, 3, 2, 3, 4, 5, 6, 7, 8, 3, 2, 4, 4, 4, 4, 3]
-            },
-            {
-                yAxis: 1,
-                data: [
-                    1,
-                    2,
-                    3,
-                    2,
-                    3,
-                    4,
-                    5,
-                    6,
-                    7,
-                    8,
-                    3,
-                    2,
-                    4,
-                    4,
-                    4,
-                    4,
-                    3
-                ].reverse()
-            }
-        ]
+        series: [{
+            data: [1, 2, 3, 2, 3, 4, 5, 6, 7, 8, 3, 2, 4, 4, 4, 4, 3]
+        }, {
+            yAxis: 1,
+            data: [6, 7, 8, 3, 2, 4, 4, 4, 4, 3, 3, 2, 4, 4, 4, 4, 3]
+        }]
     });
 
     assert.ok(
         chart.annotations[0].shapesGroup.getBBox().y === chart.yAxis[1].top,
         'Annotation measure should be visible on vary yaxis (#13664).'
     );
+
+    assert.close(
+        chart.annotations[0].shapesGroup.getBBox().y,
```

Comment:
I would assign this to a variable in order to reduce the number of times when this method is fired.